lib/stencil-init.js

```diff
@@ -115,8 +133,13 @@ class StencilInit {
                         return true;
                     }
                 },
-            },
-        ]);
+            });
+        }
+
+        return { 
+            ...await this.inquirer.prompt(prompts),
```

Comment:
We are calling `inquirer.prompt` anyway even if all the answers are provided as CLI options, but the idea of the PR was (if I understand it correctly) to avoid calling inquirer.prompt.

Reply:
Yeah, you're right. Let me take another look. :)